src/ol/view.js

```diff
@@ -273,15 +275,17 @@ ol.View.prototype.getHints = function() {
  * @api stable
  */
 ol.View.prototype.calculateExtent = function(size) {
-  goog.asserts.assert(this.isDef(),
```

Comment:
getState is now the only function using isDef, we might as well get rid of the isDef function in that case and handle it in getState?